finetune/base_models/tcn/model.py

```diff
@@ -0,0 +1,35 @@
+import os
+from urllib.parse import urljoin
+
+from finetune.base_models import SourceModel
+from finetune.base_models.gpt.encoder import GPTEncoder
+from finetune.base_models.gpt2.encoder import GPT2Encoder
+from finetune.base_models.tcn.featurizer import tcn_featurizer
+from finetune.util.download import GPT2_BASE_URL, FINETUNE_BASE_FOLDER
+
+
+class TCNModel(SourceModel):
+    is_bidirectional = False
+    encoder = GPT2Encoder
+    featurizer = tcn_featurizer
+    settings = {
```

Comment:
Important that we set n_epochs to a higher value for TCNModel since we initialize from random.  Maybe 5 to start?